tests/unit/examples/test_z_legacy_notebooks.py

```diff
@@ -85,75 +85,6 @@ def run_triton_server(modelpath):
 
 # pylint: disable=unused-import,broad-except
```

Comment:
we keep the test_z_legacy_notebooks for a specific purpose? do we still need that unit test? I am asking not that I know the answer :)

Reply:
This file came from NVT and had a bunch of example notebook tests in it, but I'm not sure if we still need it either. I removed the tests that looked directly related to the MovieLens example, but if we remove more of the example notebooks maybe we can remove the rest of this too?